skbio/stats/ordination/_principal_component_analysis.py

```diff
@@ -0,0 +1,207 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+from scipy.linalg import eigh, svd
+from scipy.sparse.linalg import svds
+from ._ordination_results import OrdinationResults
+from skbio.table._tabular import _create_table, _create_table_1d, _ingest_table
+
+
+def pca(
+    X,
+    method="svd",
+    dimensions=None,
+    sample_ids=None,
+    feature_ids=None,
+    output_format=None,
+):
+    r"""Perform Principal Component Analysis (PCA).
+
+    Principal component analysis (PCA) is a dimensionality reduction technique
+    that finds the directions (principal components) in feature space which
+    maximize the variance among samples. The original samples are projected
+    onto these directions to obtain a lower-dimensional representation of the
+    data that captures as much variance as possible.
+
+    PCA operates on a set of *n* samples that are each associated with a set of
+    *p* features. Each sample is then a row of the *n*x*p* data matrix *X*,
+    whose columns are the features of the data.
+    The goal of PCA is to find directions **w**_i in feature space along which
+    the variance of the samples is maximized, the so-called principle components
+    of the data matrix:
+    max_**w**_i var(X_centered**w**_i)
+
+    The principle components of *X* are the unit eigenvectors of the
+    covariance matrix:
+    C = 1/(n-1) X_centered^T X_centered    < can use sigma_XX instead
```

Comment:
This should be a nicely rendering Latex equation. Use of `X_centered` (which is a nice Python variable) might be replaced with mathematical symbols.